content/docs/cdn-links.md

```diff
@@ -1,37 +1,37 @@
 ---
 id: cdn-links
-title: CDN Links
+title: CDN लिंक
 permalink: docs/cdn-links.html
 prev: create-a-new-react-app.html
 next: hello-world.html
 ---
 
-Both React and ReactDOM are available over a CDN.
+React एवं ReactDOM दोनों CDN पर उपलब्ध हैं। 
 
 ```html
 <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
 <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
 ```
 
-The versions above are only meant for development, and are not suitable for production. Minified and optimized production versions of React are available at:
+उपरोक्त वर्शन केवल परीक्षण परिवर्धन के लिए हैं, एवं परिनियोजन के लिए नहीं। React के minified एवं optimized परिनियोजन वर्शन निम्नलिखित लिंक पर उपलब्ध हैं:
```

Comment:
`optimized` => `ऑप्टीमाइज़्ड`

Reply:
Hi @arshadkazmi42, would you want me to transliterate `minified` as well? minified => मिनीफ़ाइड ?

Reply:
@AninditaBasu Let's keep `minified` as is, as its react related word.